src/pages/comparison/interfaces/filter.interface.ts

```diff
@@ -0,0 +1,27 @@
+export interface InpSwitch {
```

Comment:
Don't use shortcuts... It is hard to grasp for other developers what did you mean